jahorwitz/slopopedia
# Feature/slop 42/image uploads for slop submissions

## Describe your changes

**RELATED BACK END PR:** [HERE](https://github.com/jahorwitz/slopopedia-api/pull/20)

Refactored numerous components to reflect changes outlined in the corresponding backend pr.

### Testing Instructions
1. Run backend branch mentioned above.
2. `git switch feature/slop-42/image-uploads-for-slop-submissions` 
3. test submit slop page.
4. view home page to confirm submission.

**SCREENSHOT:** 
![Selection_014](https://github.com/jahorwitz/slopopedia/assets/108188726/ccc6efe2-5d42-4540-a49f-e390d281995e)

## Issue ticket number and link
https://tripleten-apiary.atlassian.net/browse/SLOP-42?atlOrigin=eyJpIjoiMjJjYjdlNTU1MTljNGIxMDgxMjkzNzQ5MjM3YTljYzIiLCJwIjoiaiJ9

## Checklist before requesting a review

- [ ] If there are changes to components, I have added / updated automated tests
- [ ] I have made any necessary updates to Storybook to accompany these changes
